osx/setup-db.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Sets up the SolarNetwork PostgreSQL DB
+
+WORKSPACE=$1
+
+# Make sure that a workspace has been specified
+if [ -z "$WORKSPACE" ]; then
+  "A SolarNetwork workspace must be specified"
+  exit 1
+fi
+
+# Check that PostgreSQL is installed
+type -P psql &>/dev/null && echo "Configuring psql"  || { echo "$psql command not found."; exit 1; }
+
+# Set up the PostgreSQL database
+dropdb solarnetwork
+dropuser solarnet
+
+dropdb solarnet_unittest
+dropuser solarnet_test
+
+createuser -AD solarnet
+psql -U postgres -d postgres -c "alter user solarnet with password 'solarnet';"
+createdb -E UNICODE -l C -T template0 -O solarnet solarnetwork
+createlang plv8 solarnetwork
+psql -U postgres -d solarnetwork -c "CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA public;"
```

Comment:
Here's where you need to add another `CREATE EXTENSION` call for the `pgcrypto` extension.